stdlib/@tests/stubtest_allowlists/linux.txt

```diff
@@ -2,8 +2,6 @@
 # TODO: Allowlist entries that should be fixed
```

Comment:
Can remove this comment too since the section is now empty

Reply:
Sure